charms/knative-serving/src/manifests/KnativeServing.yaml.j2

```diff
@@ -7,6 +7,10 @@ metadata:
 spec:
   version: {{ serving_version }}
   config:
+  {% if queue_sidecar_image %}
```

Comment:
Based on my comment on `config.yaml`, this would change to `if custom_images` instead.